src/script/target/webpack/getConfiguration.js

```diff
@@ -0,0 +1,18 @@
+const overrideConfiguration = require('./overrideConfiguration')
+const getRaw = require('./getRaw')
+const setEnv = require('./setEnv')
+
+module.exports = async (state, callbacks) => {
```

Comment:
I try to find a pattern that scale to all our functions.
Given the fact that you can pass multiple callbacks to a function, I wrote `callbacks` parameter as the second one.